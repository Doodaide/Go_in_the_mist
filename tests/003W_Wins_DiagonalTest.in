place S1
place C9
place S3
place D10
place S5
place E11
place S7
place F12
place G13
place B8