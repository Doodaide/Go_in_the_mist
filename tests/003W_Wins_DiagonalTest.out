White wins!
S1C9S3D10S5E11S7F12G13B8
Thank you for playing!
